lib/Safari/SafariDriver.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace Facebook\WebDriver\Safari;
+
+use Facebook\WebDriver\Exception\WebDriverException;
+use Facebook\WebDriver\Remote\DesiredCapabilities;
+use Facebook\WebDriver\Remote\DriverCommand;
+use Facebook\WebDriver\Remote\RemoteWebDriver;
+use Facebook\WebDriver\Remote\Service\DriverCommandExecutor;
+use Facebook\WebDriver\Remote\WebDriverCommand;
+
+class SafariDriver extends RemoteWebDriver
+{
+    /**
+     * @return static
+     */
+    public static function start(DesiredCapabilities $desired_capabilities = null, SafariDriverService $service = null)
+    {
+        if ($desired_capabilities === null) {
+            $desired_capabilities = DesiredCapabilities::safari();
+        }
+        if ($service === null) {
+            $service = SafariDriverService::createDefaultService();
+        }
+
+        $executor = new DriverCommandExecutor($service);
+        $driver = new static($executor, null, $desired_capabilities);
+        $driver->startSession($desired_capabilities);
+
+        return $driver;
+    }
+
+    public function startSession(DesiredCapabilities $desired_capabilities)
+    {
+        $command = new WebDriverCommand(
+            null,
+            DriverCommand::NEW_SESSION,
+            [
+                'capabilities' => [
+                    'firstMatch' => [(object) $desired_capabilities->toW3cCompatibleArray()],
+                ],
```

Comment:
Does SafariDriver even support OSS protocol? Maybe we can only support the W3C one?

Reply:
Actually Safari < 12 support OSS and GitHub Actions support macOS-11,
Safari >= 12 support only W3C and it's GitHub Actions support macOS-latest